restapi/pom.xml

```diff
@@ -150,6 +150,12 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.reactivex.rxjava2</groupId>
+      <artifactId>rxjava</artifactId>
+      <version>2.2.15</version>
+      <scope>compile</scope>
```

Comment:
are we certain this is compile only?

Reply:
Good catch! Thx! (it was auto-added by IntlliJ and I did not doule-check :facepalm: )

Reply:
fixed